### PR TITLE
Added more details to README.md

### PR DESCRIPTION
**Pull request by MisterUmba**
Added more details that shines a light on what this project is about.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 ## Tic Tac Toe
 
 - What is this project?
+This project was build for the black boys code organization. It's intetions is to be used for teaching fundamental ideas in building a website.
```
Comment (delisma) [inline README.md]:
```suggestion
This project was build for the Black Boys Code organization. Its intentions is to be used for teaching fundamental ideas in building a website. 
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
 ## Tic Tac Toe
 
 - What is this project?
+This project was build for the black boys code organization. It's intetions is to be used for teaching fundamental ideas in building a website. 
 - How does it work?
+Very simple. Open the index.html page in your browser.
```
Comment (delisma) [inline README.md]:
```suggestion
Very simple, go to [](https://black-boys-code.github.io/TicTacToe). Or open this repo in Glitch project.
)```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
 ## Tic Tac Toe
 
 - What is this project?
+This project was build for the black boys code organization. It's intetions is to be used for teaching fundamental ideas in building a website. 
 - How does it work?
+Very simple. Open the index.html page in your browser. 
 - Who will use this project?
+Students learning the basics of web development.
```
Comment (delisma) [inline README.md]:
```suggestion
Black Boys Code students learning the basics of web development. 
```

**Review by delisma — CHANGES_REQUESTED** [review]

**Comment by MisterUmba** [comment]
I've made all the requested changes.

**Pull request merged**